client/lib/menu-data/menu-data.js

```diff
@@ -33,13 +34,18 @@ const sites = sitesFactory();
  */
 var DEFAULT_MENU_ID = 0;
 var HOMEPAGE_MENU_ITEM_ID = -99;
+var NEWPAGE_MENU_ITEM_ID = -100;
```

Comment:
No, there is no importance, but we need to avoid colliding with the values saves as JSON in wp-admin database.
Each menu item in DB has integer id to tie it into the actual item it represents.
Before saving I'm replacing -100 with an ID of proper page.
I chose -100 to keep the pattern in this file..